packages/nimble-components/src/select/tests/select.stories.ts

```diff
@@ -34,10 +37,14 @@ const metadata: Meta<SelectArgs> = {
         },
         actions: {
             handles: ['change']
+        },
+        toolbar: {
+            zoom: { hidden: true }
```

Comment:
I thought you were trying to only disable this in the docs view? Was there a change in direction? If you are always disabling the zoom toolbar does the "popupFixstyling" snippet still need css to disable the buttons?

Reply:
Yes, I was wrong about the zoom on the other pages not being affected. This zoom hidden true config does not affect the doc page, so we still need the CSS-based hiding for that, too.